src/olympia/lib/es/management/commands/reindex.py

```diff
@@ -68,23 +68,7 @@ def update_aliases(actions, stdout=sys.stdout):
 def create_new_index(alias, new_index, stdout=sys.stdout):
     log('Create the index {0}, for alias: {1}'.format(new_index, alias),
         stdout=stdout)
-
-    config = {}
-
-    # Retrieve settings from last index, if any
-    if ES.indices.exists(alias):
-        res = ES.indices.get_settings(alias)
-        idx_settings = res.get(alias, {}).get('settings', {})
```

Comment:
This never worked because `get_settings` returns a dict with the real index name as the key, not the alias.